sympy/core/power.py

```diff
@@ -1212,6 +1212,22 @@ def _is_one(expr):
                 or self.base.is_irrational):
                 return self.exp.is_rational
 
+    def _eval_is_transcendental(self):
+        # Gelfond-Schneider theorem
+        def _is_one(expr):
+            try:
+                return (expr - 1).is_zero
+            except ValueError:
+                # when the operation is not allowed
+                return False
+
+        if self.base.is_zero or _is_one(self.base):
+            return False
+        elif self.base.is_real and self.exp.is_irrational:
+            return True
+        elif self.base.is_transcendental and self.exp.is_zero == False:
```

Comment:
This is also wrong. The theorem only applies when the base is algebraic (see https://en.wikipedia.org/wiki/Gelfond%E2%80%93Schneider_theorem, the counterexample `(sqrt(2)**sqrt(2))**sqrt(2)`. This would be true if you added `self.exp.is_rational` as `t**r` is transcendental for transcendental `t` and nonzero rational `r`.

Reply:
Yes, I also figured this, will make the required changes in next round. Thanks,